identity_presentation/src/commonMain/kotlin/net/thechance/mena/identity/presentation/screen/pickLocation/PickLocationScreen.kt

```diff
@@ -0,0 +1,115 @@
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.statusBarsPadding
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import cafe.adriel.voyager.navigator.Navigator
+import mena.identity_presentation.generated.resources.Res
+import mena.identity_presentation.generated.resources.confirm
+import mena.identity_presentation.generated.resources.pick_location_title
+import net.thechance.mena.designsystem.presentation.component.button.PrimaryButton
+import net.thechance.mena.designsystem.presentation.component.scaffold.Scaffold
+import net.thechance.mena.designsystem.presentation.theme.theme.Theme
+import net.thechance.mena.identity.presentation.base.BaseScreen
+import net.thechance.mena.identity.presentation.components.AuthAppBar
+import net.thechance.mena.identity.presentation.components.ErrorSnackBar
+import net.thechance.mena.identity.presentation.screen.enableLocationScreen.EnableLocationScreen
+import net.thechance.mena.identity.presentation.screen.pickLocation.components.EditMapButton
+import net.thechance.mena.identity.presentation.screen.pickLocation.components.GpsFabButton
+import net.thechance.mena.identity.presentation.screen.pickLocation.components.Map
+import org.jetbrains.compose.resources.stringResource
+
+class PickLocationScreen() : BaseScreen<PickLocationScreenViewModel,
+        PickLocationScreenUIState,
+        PickLocationScreenUIEffect,
+        PickLocationScreenInteractionListener>() {
+
+    @Composable
+    override fun Content() {
+        InitScreen(getScreenModel())
+    }
+
+    @Composable
+    override fun OnRender(
+        state: PickLocationScreenUIState,
+        listener: PickLocationScreenInteractionListener
+    ) {
+        Scaffold(
+            topBar = {
+                AuthAppBar(
+                    title = stringResource(Res.string.pick_location_title),
+                    onClickBack = listener::onClickBack,
+                    backgroundColor = Theme.colorScheme.background.surfaceLow
+                )
+            }
+        ) {
+            Map(
+                cameraPosition = state.cameraPosition,
+                onCameraMoved = listener::onMoveCamera,
+                onMapClick = listener::onClickMap,
+                anchorLocation = state.pointerLocation,
+                isLocked = state.isMapLocked,
+                currentLocation = state.currentLocation,
+                animateToCurrentLocation = state.animateToCurrentLocation
+            ) {
+                Column(
+                    Modifier
+                        .padding(16.dp)
```

Comment:
For following design padding values
```suggestion
                        .padding(Theme.spacing._16)
```